### DISCUSSION TITLE
Larger accessibility type sizes

### DISCUSSION THREAD

**Discussion by jamesholcomb** [jpudysz/react-native-unistyles#148] [Feature Requests]
Hi Jacek, are there plans to implement content size categories for the larger accessibility types on iOS and Android?  It seems that 'unspecified' is returned if enabled in the os.

refs:

https://developer.apple.com/design/human-interface-guidelines/typography#iOS-iPadOS-larger-accessibility-type-sizes
https://developer.android.com/reference/android/util/TypedValue#applyDimension(int,%20float,%20android.util.DisplayMetrics)

**Comment by jpudysz** [comment]
Hey, for iOS I'm using content size category: https://developer.apple.com/documentation/uikit/uicontentsizecategory

I already mapped all values. For Android there is no counterpart, so I'm using font scale as noted in docs. 

Do you need any other values ? What's your use case?

**Comment by jamesholcomb** [comment]
In ios, if you enable "Larger Accessibility Sizes" in Accessibility settings, a value of 'unspecified' is returned.  For android, "huge" is always returned if scale > 1.3 per your map (5 content sizes for android), but wondering if more granular sizing similar to iOS could be achieved.

**Reply by jpudysz** [reply]
Yeah I agree that some improvements are necessary. I will take a look at it in the upcoming days.

**Comment by jpudysz** [comment]
Added to 2.4.0 roadmap
